mkt/extensions/migrations/0002_extension_add_uuid.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can't take the credit for it, the idea comes from the django docs. I was about to write a single python migration, the same kind that would sometimes break back when we were using schematic. Was pretty happy to read how to do it properly in the docs :)